Lushu/src/main/kotlin/lushu/ContextGrammar/Grammar/DSL.kt

```diff
@@ -0,0 +1,155 @@
+package lushu.ContextGrammar.Grammar
+
+class DSL(
+    val tags: MutableList<String> = mutableListOf("<c>", "<s>", "<*>", "<t>"),
```

Comment:
```suggestion
    val tagNames = listOf("c", "s", "*", "t"),
    val tags = tagNames.map { it -> "<${it}>" },
```
We don't need to repeat "<" and ">" everywhere here.

Reply:
Actually, we should create an Enum class out of this. Please see https://kotlinlang.org/docs/enum-classes.html for examples on how to do this.